### DISCUSSION TITLE
connect to internet with a VPN (netmod - windows)

### DISCUSSION THREAD

**Discussion by mwdiss** [rejetto/hfs#789] [Q&A]
i cant connect to internet cuz im using a vpn(v2ray) to connect to the internet {tunnel (:public), unless i dont have internet}.
my router ip is 10.0.0.1 (home), but HFS detects my vpn's ip :
![image_2024-10-27_195856017](https://github.com/user-attachments/assets/98a527a4-7428-4d98-a531-8e61e1d570a5)
![image_2024-10-27_200755175](https://github.com/user-attachments/assets/d6dd2f3f-5384-4345-bad1-60e033cf88bb)

![image_2024-10-27_201100217](https://github.com/user-attachments/assets/6d886cd7-eaf6-46b3-8c77-847fd6d3613c)

am i only able to share files on my LAN forever :( 
any suggestions to fix this (or alt methods access a folder from anywhere on my pc) ?

**Comment by rejetto** [comment]
did you click on "double NAT" and read what's adviced there?

also, what version are you using?

**Comment by rejetto** [comment]
i just wrote this page, I hope it help.
https://github.com/rejetto/hfs/wiki/Work-on-the-internet#double-nat
